src/chip.tsx

```diff
@@ -56,9 +58,22 @@ type ClickableChipProps = ExclusifyUnion<HrefChipProps | ToChipProps | OnPressCh
 type ChipProps = ExclusifyUnion<SimpleChipProps | ClosableChipProps | ToggleChipProps | ClickableChipProps>;
 
 const Chip: React.FC<ChipProps> = (props: ChipProps) => {
-    const {Icon, children, id, dataAttributes, active, onClose} = props;
+    const {Icon, children, id, dataAttributes, active, badge, onClose} = props;
     const {texts, isDarkMode} = useTheme();
     const overAlternative = useThemeVariant() === 'alternative';
+    const {isMobile} = useScreenSize();
+
+    const paddingLeft = Icon && isMobile ? 16 : 8 && Icon ? 8 : 20 && isMobile ? 20 : 12;
+    const paddingRight = isMobile ? 20 : 12;
+    const paddingIcon = isMobile ? 16 : 8;
+    const badgeSize = badge === 9 + 1 ? 24 : 16;
+
+    const renderBadge = () => {
```

Comment:
I found a weird behavior when you click in a badge. Why badge has this interaction? Badge is not clickable.

https://github.com/Telefonica/mistica-web/assets/6722153/1759def5-167c-4bc5-8a99-480adc72fa21

Reply:
now the problems doesn't exists but the badge has cursor pointer. Badge doesn't have interaction.. the cursor pointer has not to apply